README.md

```diff
@@ -13,8 +13,9 @@ GraphQL endpoint providing a single endpoint to access data from Argo's many ser
 
 - Set up environment: copy `.env.schema` to `.env` and update environment accordingly if needed. Values provided in the schema file can be used when running the server locally for development.
 - Install dependencies: `npm ci` (depending on your Node version, it may require a '--legacy-peer-deps' flag)
-- Navigate to `./compose` and run `docker-compose up -d` to start up elasticsearch
-- `npm run programDashboardEsInit && npm run fileCentricEsInit` to initialize some data to elasticsearch for local development
+- Insure your system has Docker and Postgres installed and running.
+- Navigate to `./compose` and run `docker-compose up -d` to start up elasticsearch & program service
+- Navigate back to project root and run `npm run programDashboardEsInit && npm run fileCentricEsInit` to initialize some data for local development.
```

Comment:
ElasticSearch is failing for me at this step with a `429` error

Reply:
This may be a MacOS issue, working for me after update to Ventura 13.4

Reply:
There's a specific ES image needed for M1 macs and the wrong one is used here. I'll update to make sure we have the right one